e2e-tests/cypress/support/helpers/SmartKbnClient.ts

```diff
@@ -0,0 +1,62 @@
+export class SmartKbnClient {
```

Comment:
Why this name is better than the previous one? IMO the intentions were clear, make a direct request to the Kibana

Reply:
the problem with the previous one was that some of the functions were an advanced usage of API (eg. call get objects and for each object call delete). 

But I agree that this name doesn't tell much. I've changed it to KbnApiClient and KbnApiAdvancedClient. The first one is the wrapper around KBN REST API and the second one contains methods that deliver some more functionality than just wrapping the API.

please take a look and say what do you think

Reply:
That's an interesting approach, at first I felt that this division is too much, but maybe it makes sense. Looks good, let's check this approach